R/score.R

```diff
@@ -0,0 +1,57 @@
+# score.R
+
+#' Produce a gH object: a hash of genes and their associated scores
+#' (likelihood that observed mutations exceed background mutations at FDR of 1%)
+#'
+
+score <- function(gX, gH, silent = FALSE, noLog = FALSE) {
+    
+    check parameters with .checkArgs()
+    
+    open gX file
+    
+    # for each gene, check if it already in the gH hash and if yes, do not calculate its score (in case have a repeat of a gene in gX, would just use first calculated score)
+    
+    for gene in gX {
+        if gene already in gH {
+            break
+        } else {
+    
+            # define background mutation rate for gene
+            back_mut = (missense + nonsense)/silent   # would be more accurate to use the background mutation data from Samocha et al., 2013 (http://www.nature.com/ng/journal/v46/n9/full/ng.3050.html)

```

Comment:
A: how is ```(missense + nonsense)/silent``` a background mutation rate?

B: Can you calculate a score according to Samocha *et al.* with the data you **have**? If so, how?